### ISSUE TITLE
VAMC Facility template: adopt V1 and V3 components where components are not used

### ISSUE THREAD

**Issue by jilladams** [department-of-veterans-affairs/va.gov-cms#17654]
## Description
VAMC Facility pages need the following updates to components:

Example pages:

- [Jamaica Plain VA Medical Center](https://www.va.gov/boston-health-care/locations/jamaica-plain-va-medical-center/)
- [Minneapolis VA Medical Center](https://www.va.gov/minneapolis-health-care/locations/minneapolis-va-medical-center/)

### Elements that are not using components that should be
1. Link
   - Links are not using a component but should be using [the v1 component](https://design.va.gov/components/link/) if possible, with the understanding that CMS content has limitations
2. ~Telephone~
   - ~[The component](https://design.va.gov/components/telephone) is not being used anywhere but should be if possible, with the understanding that CMS content has limitations~
   - ~The `aria-describedby` [variant](https://design.va.gov/components/telephone#aria-described-by) should be used for the main and mental health numbers~
   - ~The standard variant should be used in other places~
   - ~Exts in phone numbers should be parsed correctly~
3. ~Alerts~
    - ~The Operating status alerts should be using the [slim alert component](https://design.va.gov/storybook/?path=/docs/uswds-va-alert--default#slim) with an bold [link component](https://design.va.gov/storybook/?path=/docs/components-va-link--default) inside it. This was already done on the Locations template~
4. Back to top
   - The page meets the criteria guidelines for the [Back to top component](https://design.va.gov/components/back-to-top) I believe so this should be added
5. Service Location
   - Update Service Locations to use [the card component](https://design.va.gov/components/card#default---white-with-border).
6. Related Links/Spotlight
   - There is a list of links that we are going to change to align with the DS. This is going to use [Related Links](https://design.va.gov/components/link/collection#related-links). Based on Storybook, I don't believe there is standard component markup for this (i.e. `<va-XXX>`) [Here is the annotated Figma design](https://github.com/department-of-veterans-affairs/va.gov-cms/issues/16952) for what this will look like:
 ![Screenshot 2024-03-22 at 1 54 08 PM](https://github.com/department-of-veterans-affairs/va.gov-cms/assets/106678594/9eb2077d-1501-4099-8f8a-a6956f2db05b)

### NOTES
* #16963 tracks V3 adoption for existing V1 components
* V1 components without V3 equivalents (Icebox)
1. On this page: Currently using [v1 component](https://design.va.gov/components/on-this-page) which is the most up to date component

## Acceptance Criteria
- [x] Link component has been added
- [x] Slim alerts with bold link have been added for operating status
- [x] Service locations use card component
- [x] Related Links have been added
- [x] Back to top component has been added
- [x] Update and run existing test(s).
- [x] Add new tests as necessary to cover the changes OR cut a ticket to represent the new tests, if they require additional scope
- [x] Requires accessibility review
- [ ] When PR is approved and ready to merge, tag @jilladams for change management
- [ ] Change management for Related links component

**Comment by laflannery** [comment]
@jilladams and @mmiddaugh There is a [facility with a Mental health number with an ext](https://www.va.gov/eastern-oklahoma-health-care/locations/yale-avenue-va-clinic/). because this is not using the component, the extension is not being parsed correctly and therefore this cannot easily be called directly from mobile phones by just tapping on it. This was reported by the Help desk in [this slack thread](https://dsva.slack.com/archives/CDHBKAL9W/p1711648596252219). For this reason I think this would be a good ticket to consider as one of the first to tackle after the priority v3 component tickets

**Comment by jilladams** [comment]
Noted for refinement.

**Comment by jilladams** [comment]
@laflannery - based on our Q2 priorities, dealing with all impostor components / adoption isn't our top priority. Do you feel comfortable with me breaking out the telephone piece here as the a11y #2 bug, so we can laser fix that, and leave the rest of the adoption for when we're able to prioritize other DS components?

**Comment by jilladams** [comment]
Laura confirmed: https://github.com/department-of-veterans-affairs/va.gov-cms/issues/17745

**Comment by laflannery** [comment]
Telephone moved to #16148

**Comment by jilladams** [comment]
@randimays @jtmst we have 2 Facilities components tickets that we haven't had a chance to refine or point yet. Since y'all are doing this work, maybe you wouldn't mind pointing?  It's here and on #17657

**Comment by jilladams** [comment]
@jilladams @laflannery to use the Operating status audit to help identify a facility that has a published Operating Status alert, in order to give an example for FE to reference.

**Comment by jilladams** [comment]
@eselkin flagging that Public Websites is going to help Facilities in Sprint 4 with additional component adoptions, which will now start to include Service Locations. When you review code for these, and Service Locations are implicated, just a flag for you to have eyes on that, specifically.

**Comment by laflannery** [comment]
In a shocking twist - the Operating status alerts have already been updated! Confirmed all 2 statues:
- [Closed](https://www.va.gov/charleston-health-care/locations/market-commons-va-clinic/)
- [Limited Services and hours](https://www.va.gov/charleston-health-care/locations/myrtle-beach-va-clinic/)
- [Facility notice](https://www.va.gov/charleston-health-care/locations/charleston-va-clinic/)
They must have been included in the Locations update

**Comment by jilladams** [comment]
BLESS

**Comment by laflannery** [comment]
@thejordanwood and @randimays I don't know if this helps but this is [VA's Icon page](https://design.va.gov/foundation/icons) that seems to have old FA icons compared to the new icons and which would apply

**Comment by randimays** [comment]
Thank you @laflannery! I had in my notes somewhere that the USWDS site was the source of truth but I think you're right; it's actually that VA Icon page.

**Comment by randimays** [comment]
The change management conversation has been started in Slack here:https://dsva.slack.com/archives/C06NMUK12LR/p1716323803156549 Please let me know if there is anything else needed from an engineering perspective.

Changes have been verified successfully in production. Closing!
